src/apps/intermediate-list/fee-list/fee-list.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from "react";
+import { FC } from "react";
+import ListHeader from "../../../components/list-header/list-header";
+import { FeeV2 } from "../../../core/fbs/model";
+import { useText } from "../../../core/utils/text";
+import FeeListItem from "../fee-list-item/fee-list.item";
+import TotalPaymentPay from "../stackable-fees/total-payment-pay";
+
+interface FeeListProps {
+  openDetailsModalClickEvent: (faustId: string) => void;
+  itemsPrePaymentChange: FeeV2[] | null;
+  itemsPostPaymentChange: FeeV2[] | null;
+  totalFeePrePaymentChange: number;
+  totalFeePostPaymentChange: number;
+}
+const FeeList: FC<FeeListProps> = ({
+  openDetailsModalClickEvent,
+  itemsPrePaymentChange,
+  itemsPostPaymentChange,
+  totalFeePrePaymentChange,
+  totalFeePostPaymentChange
+}) => {
+  const t = useText();
+
+  return (
+    <div>
+      {!itemsPrePaymentChange && !itemsPostPaymentChange && (
+        <>
+          <ListHeader header={<>{t("unpaidFeesText")}</>} amount={0} />
+          <div className="dpl-list-empty">{t("emptyIntermediateListText")}</div>
+        </>
+      )}
+      {itemsPrePaymentChange && (
+        <div>
+          <ListHeader
+            header={
+              <>
+                {t("unpaidFeesText")} -{" "}
+                <b>{t("prePaymentTypeChangeDateText")}</b>
```

Comment:
I am not going to complain about it, but this is the only place that I have seen the b tag used. Is it not possible to solve via a class?